### PR TITLE
Updated castlevania.yaml

### PR DESCRIPTION
**Pull request by eldri7ch2**
## Description

Updated and corrected information about Symphony of the Night Randomizers.

### CODE REVIEW

**Comment by eldri7ch2** [comment]
I am so lost. Somehow I still had some of the issues?

**Comment by eldri7ch2** [comment]
BTW, front-end coding is not my forté

**Comment by Die4Ever** [comment]
you can access the results of the automation by clicking the Details, or just run the validation script on your own computer

`Castlevania.yml: undefined games in SOTN Area Rando (SOTN AR) : Castlevania: Symphony of the Night (PlayStation / Saturn / PSP)`

that's not a real game title, ditch the list of platforms like I did on your other pull request

**Review by eldri7ch2 — COMMENTED** [review]
Moved platforms to Identifiers

**Review by eldri7ch2 — COMMENTED** [review]
Learning process. My bad.

**Review by eldri7ch2 — COMMENTED** [review]
One day I will make something valid.

**Comment by Die4Ever** [comment]
`Castlevania.yml: unused games: Castlevania: Symphony of the Night (PlayStation / Saturn)`

so it needs to be removed from the games list at the top

**Comment by eldri7ch2** [comment]
Not there when I view it:

image

**Comment by eldri7ch2** [comment]
When I do the file reviews, am I viewing a different file? If so, how do I replace what is being reviewed?

**Comment by Die4Ever** [comment]
line 47

**Review by eldri7ch2 — COMMENTED** [review]
removed console list from line 47

**Comment by eldri7ch2** [comment]
That made no sense, I didn't even edit that line. Anyway, now it passed.

File: src/series/Castlevania.yml [hunk 1]
```diff
@@ -86,25 +82,35 @@ randomizers:
...
     added-date: 1900-01-01
 -   games:
-    - 'Castlevania: Symphony of the Night (PlayStation / Saturn)'
-    identifier: 3snowp7im's Area Randomizer
-    url: https://github.com/3snowp7im/SotN-Randomizer
-    comment: 'Includes a Richter converter for Wild Mouse''s item randomizer, and a standalone Maria item randomizer. Browser version here: <https://sotn.io/>'
-    updated-date: 2024-08-21
+    - 'Castlevania: Symphony of the Night'
+    identifier: "Castlevania: SOTN Randomizer"
+    url: https://symphonyrando.fun
```
Comment (Die4Ever) [inline src/series/Castlevania.yml:87]: this url doesn't work https://symphonyrando.fun

Comment (eldri7ch2) [reply src/series/Castlevania.yml:87]: Needs the www after //. My bad, i forget that all the time because it was a recent requirement change

Comment (Die4Ever) [reply src/series/Castlevania.yml:87]: seems like the actual URL is https://www.symphonyrando.fun/ with the www

**Review by eldri7ch2 — COMMENTED** [review]
added www.

File: src/series/Castlevania.yml (hunk 1 above)
Comment (eldri7ch2) [reply src/series/Castlevania.yml]: Yeah

File: src/series/Castlevania.yml [hunk 2]
```diff
@@ -86,25 +82,35 @@ randomizers:
...
 -   games:
     - 'Castlevania: Symphony of the Night'
-    identifier: Wild Mouse's
-    url: https://sotn.io/
+    identifier: "SOTN X-Box Rando"
+    url: https://github.com/MottZilla/SOTN_XB_RANDO
+    comment: "MottZilla's Randomizer for the XBLA version of SOTN Includes many of the same presets as the PlayStation's community version. Only works with the Canary build of the Xenia emulator or on Modded X-Box 360's."
+    updated-date: 2024-12-03
+    added-date: 2024-12-03
+    opensource: true
```
Comment (Die4Ever) [inline src/series/Castlevania.yml:106]: https://github.com/MottZilla/SOTN_XB_RANDO does not appear to include source code for the randomizer

```suggestion
```

Comment (eldri7ch2) [reply src/series/Castlevania.yml:106]: O, damn. I thought it did. Maybe he didn't want his spaghetti code all over. Lemme fix that.

**Review by eldri7ch2 — COMMENTED** [review]
Removed opensource flag from XBLA SOTN Rando

**Comment by Die4Ever** [comment]
your updates are now live, thank you!

**Pull request merged**